### ISSUE TITLE
OSX Wwise won't generate soundbank because plugin does not have license.

### ISSUE THREAD

**Issue by unriginal** [enzienaudio/heavy#127]
Hello, I've just tried using a plugin from the Wwise export in OSX. It loads into the Wwise editor, and plays sounds from the plugin properly from inside the editor, but whenever I try to generate a soundbank I get an error saying "No license for 'wind' plug-in. Please contact sales@audiokinetic.com to obtain a license." (the plugin is called wind). I'm on OSX 10.11.3, Wwise version 2016.1.2.5863 (although I had this issue with  2016.1.1.5823 as well).

**Comment by diplojocus** [comment]
hi @unriginal could you send me a link to your plugin and I'll take a look

**Comment by diplojocus** [comment]
You're correct this does happen, and doesn't seem to make a difference if I change the name slightly. Very strange. I'll try and find out what's going wrong.

**Comment by unriginal** [comment]
Yes, the plugin link is here:
https://enzienaudio.com/h/unoriginal/wind/

I also changed the name in case wind was reserved by some other plugin, but yeah, same result.

**Comment by diplojocus** [comment]
@unriginal is it possible for you to use Wwise version 2016.1.0.5775 until we figure this out? I'm able to generate soundbanks with the plugin correctly in that version.

**Comment by diplojocus** [comment]
hi @unriginal it looks like backwards compatibility with plug-ins in the latest version of Wwise has been broken, I've been told that the next release will fix it all and to wait for that. Alternatively 2016.1.0.5775 is confirmed to be working.
